lib/Parser/rterrors.h

```diff
@@ -20,6 +20,7 @@
 #define RT_ERROR_MSG_UNUSED_ENTRY
 #endif
 
+// NOTE: The VBSERR macros should not be changed due to legacy dependencies outside of Chakra.
```

Comment:
What we want to communicate is:

* These names go beyond our project and *should not* be changed.
*The definitions (numbers, strings) are also depended on in other projects and *cannot* be changed.